src/NuGet.Core/NuGet.Packaging/PackageExtractor.cs

```diff
@@ -825,7 +825,7 @@ private static void DeleteTargetAndTempPaths(string targetPath, string targetTem
 
                             File.Move(tempNupkgMetadataFilePath, nupkgMetadataFilePath);
 
-                            logger.LogVerbose($"Completed installation of {packageIdentity.Id} {packageIdentity.Version}");
+                            logger.LogVerbose($"Completed installation of {packageIdentity.Id} {packageIdentity.Version} from {packageDownloader.Source}");
```

Comment:
@jeffkl as someone who was written code that parses NuGet's output messages, what do you think about us changing this message?

Reply:
This won't affect our current logic as we're not looking at this particular line of output.  Also, there is no expectation that logging statements stay the same since we knew it could be fragile (its just the only way to do it currently)